build.gradle

```diff
@@ -0,0 +1,158 @@
+plugins {
+    id 'com.android.application' version '8.1.1'
+}
+
+android {
+    namespace 'juloo.keyboard2'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "juloo.keyboard2"
+        minSdk 4
+        targetSdkVersion 33
+        versionCode 36
+        versionName "1.24.0"
+    }
+
+    sourceSets {
+        main {
+            manifest.srcFile 'AndroidManifest.xml'
+            java.srcDirs = ['srcs']
+            res.srcDirs = ['res']
+            assets.srcDirs = ['assets']
+        }
+    }
+
+    signingConfigs {
+        // Debug builds will always be signed. If no environment variables are set, a default
+        // keystore will be initialized by the task initDebugKeystore and used. This keystore
+        // can be uploaded to GitHub secrets by following CONTRIBUTING.md#Using the local debug.keystore on the Github CI actions
+        // in order to always receive correctly signed debug APKs from the CI.
+        debug {
+            storeFile (System.env.DEBUG_KEYSTORE ? file(System.env.DEBUG_KEYSTORE) : file("debug.keystore"))
+            storePassword (System.env.DEBUG_KEYSTORE_PASSWORD ? "$System.env.DEBUG_KEYSTORE_PASSWORD" : "debug0")
+            keyAlias (System.env.DEBUG_KEY_ALIAS ? "$System.env.DEBUG_KEY_ALIAS" : "debug")
+            keyPassword (System.env.DEBUG_KEY_PASSWORD ? "$System.env.DEBUG_KEY_PASSWORD" : "debug0")
+        }
+        // Release builds will only be signed with a dedicated key if specified. This key has to be
+        // created manually. To be used with GitHub CI, key and environment variables have to be
+        // added as secrets. See file .github/workflows/make-apk.yml for further reference.
+        release {
+            if (System.env.RELEASE_KEYSTORE) {
+                storeFile file(System.env.RELEASE_KEYSTORE)
+                storePassword "$System.env.RELEASE_KEYSTORE_PASSWORD"
+                keyAlias "$System.env.RELEASE_KEY_ALIAS"
+                keyPassword "$System.env.RELEASE_KEY_PASSWORD"
+            }
+        }
+    }
+    buildTypes {
+        release {
+            minifyEnabled true
+            shrinkResources true
+            debuggable false
```

Comment:
I'm not a fan of enabling minifyEnabled and disabling debug. The app is already small enough and these help get better stacktraces when a crash happens.